src/basic/terminal-util.c

```diff
@@ -1288,11 +1289,13 @@ int vt_restore(int fd) {
         if (r < 0)
                 q = log_debug_errno(errno, "Failed to set VT in text mode, ignoring: %m");
 
-        r = vt_reset_keyboard(fd);
-        if (r < 0) {
-                log_debug_errno(r, "Failed to reset keyboard mode, ignoring: %m");
-                if (q >= 0)
-                        q = r;
+        if (vt_verify_kbmode(fd) == 0) {
```

Comment:
this function currently debug logs about everything it does. I think it would make sense to also debug log here when we don't reset the kbd, i.e.:

```c
r = vt_verify_kbdmode(fd);
if (r < 0) 
        log_debug_errno(r, "Keyboard not in XLATE or UNICODE mode, not resetting.);
else { …
```

Reply:
Done.